### PR TITLE
[12.0][NEW] mrp_bom_multi_company

### PR DESCRIPTION
**Pull request by mileo**
Remove company_id required attribute of mrp.bom

### CODE REVIEW

**Comment by bodedra** [comment]
:-1:

**Review by dreispt — APPROVED** [review]

File: mrp_bom_multi_company/models/mrp_bom.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class MrpBom(models.Model):
+
+    _inherit = 'mrp.bom'
+
+    company_id = fields.Many2one(
+        default=lambda self: self.env['res.company']._company_default_get('mrp.bom'),
+        required=False
+    )
```
Comment (bodedra) [inline mrp_bom_multi_company/models/mrp_bom.py:14]:
```suggestion
    company_id = fields.Many2one(
        required=False
    )
```

**Review by bodedra — COMMENTED** [review]
minor suggestion.

**Pull request merged**